package.json

```diff
@@ -38,6 +38,11 @@
         "icon": "resources/dark/add-location.svg",
         "category": "Adapters"
       },
+      {
+        "title": "Restart server",
+        "category": "Adapters",
```

Comment:
Change the category to Servers.
``` "category": "Servers", ```

Reply:
Why we need to change the `"category": "servers"`? How the user will know for which extension they are restarting the server.